Swiftfin tvOS/Views/ItemView/Components/ActionButtons/SelectVersionButton.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2025 Jellyfin & Jellyfin Contributors
+//
+
+import JellyfinAPI
+import SwiftUI
+
+extension ItemView {
+
+    struct SelectVersionButton: View {
+
+        @ObservedObject
+        var viewModel: ItemViewModel
+
+        // MARK: - Body
+
+        var body: some View {
+            Menu {
+                if let playButtonItem = viewModel.playButtonItem,
```

Comment:
Just pass in the view model as well as the array of media sources so these checks don't need to be made.